cmd/approve.go

```diff
@@ -98,19 +98,28 @@ Enter 'yes' to approve all, 'no' to cancel, 'reject' to reject all, a number for
 	if scanner.Scan() {
 		s = scanner.Text()
 	}
+	if len(message) == 0 {
+		scanner = bufio.NewScanner(os.Stdin)
+		fmt.Printf(
+			"\n%s",
+			`Do you want to add a message? E.g. A Change Order number or rejection reason. This will apply for all pipelines: `)
```

Comment:
```suggestion
			`Would you like to add a message? E.g. a reference to a ticket number or rejection reason. This will apply for all pipelines: `)
```